wrapanapi/systems/rhevm.py

```diff
@@ -1005,10 +1005,10 @@ def _get_image_service(self, storage_domain_name, image_name):
 
     def import_glance_image(self, source_storage_domain_name, source_template_name,
```

Comment:
please check other repos like integration tests for usage of this method. we shouldn't break anything by such rename

Reply:
@izapolsk 
In Integration_test; we have two but not passing `async`  arg so impact :+1:  
1. https://github.com/ManageIQ/integration_tests/blob/37a7552d2428390a8f52936b7a71ea7a447a8b2a/cfme/utils/template/rhevm.py#L34

2. https://github.com/ManageIQ/integration_tests/blob/b5338f9686db6acf9e9a4ce8c48cd2fe875d3f85/scripts/template_upload_rhevm_qcow2.py#L132

I checked other like `robottelo` from `satellite-qe`. they are not using this method.